client/src/views/files/FoldersPage.vue

```diff
@@ -157,6 +157,10 @@
           />
         </div>
       </div>
+      <!-- problème -->
+      <!-- voir comment mettre à jour de façon automatique le menu d'upload -->
+      <!-- avec la fonction onFileImportState() -->
+      <file-upload-menu />
```

Comment:
The menu only being visible on the FoldersPage is weird. Especially since it's the same one for all workspaces.
You start an import in one workspace, go back to the workspace list, you don't know anymore how the upload is progressing, then go in another workspace and see the menu but it seems that you're importing files into this workspace when in reality you're importing them in the previous one.

Reply:
To avoid this behaviour, the `Import menu` can be visible on the Folders and Workspaces pages.

Reply:
we'll do this:
 - show the menu on workspaces page too
 - add the relative workspace next to the size of the file